app/data/__tests__/team.test.js

```diff
@@ -0,0 +1,27 @@
+const { Team } = require('../team');
+
+describe('Equipos', () => {
+    describe('Comprobando la declaración de un nuevo equipo', () => {
+        it('Comprobando cuando se le pasa como parámetro un string', () => {
+            const team = new Team('CaracasFC');
+            expect(team).toBeTruthy();
+        });
+
+        it('Comprobando que no lanza un error al no pasar una serie de equipos.', () => {
+            const team = new Team();
+            expect(team).toBeTruthy();
+            expect(team.teamName).toBeUndefined();
+            expect(team.listPlayers).toStrictEqual([]);
+            expect(team.teamValue).toStrictEqual(0);
+    });
+
+    });
+
+    describe('Comprobando el método getTeamName()', () => {
+        it('Comprobando que devuelva el valor correcto', () =>{
+            const team = new Team('CaracasFC');
+            expect(team.getTeamName()).toBeTruthy();
+            expect(team.getTeamName()).toStrictEqual('CaracasFC');
```

Comment:
@gabodt16 ¿Estás seguro del uso de toStrictEqual? Revisaría la docu de Jest porque creo que no es para lo que estás intentando hacer.

Yo usaría lo siguiente

```suggestion
            expect(team.getTeamName()).toBe('CaracasFC');
```